.gitignore

```diff
@@ -1,5 +1,7 @@
 src/qinfer/version.py
 
+*.sublime-workspace
+
```

Comment:
I've started to address this in 7a9a8dd by making a new ``devguide/`` chapter of the docs with a section on IDE and editor integrations.